File: app/src/main/java/com/hfad/starbuzz/Java/MainActivity.java
package com.hfad.starbuzz.Java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.Spinner;

import com.hfad.starbuzz.R;

public class MainActivity extends AppCompatActivity {

    String[] colors = new String[]{"Красный", "Оранжевый", "Желтый", "Зеленый", "Синий"};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Spinner spinner = (Spinner) findViewById(R.id.spinner);
        ArrayAdapter<String> adapter = new ArrayAdapter<>(
                this,
                android.R.layout.simple_spinner_item,
                colors
        );
        spinner.setAdapter(adapter);
    }
}

File: app/src/main/java/com/hfad/starbuzz/Kotlin/MainActivity.kt
package com.hfad.starbuzz.Kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import com.hfad.starbuzz.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    val colors = arrayOf("Красный", "Оражевый", "Желтый", "Зеленый", "Синий")

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val spinner = spinner
        val adapter = ArrayAdapter(this@MainActivity, android.R.layout.simple_spinner_item, colors)
        spinner.adapter = adapter
    }
}
